src/utils/read.js

```diff
@@ -7,16 +7,10 @@ const fs = require("fs");
  * var read = require('./helpers/fsExists')({});
  * read('.babelrc'); // file contents...
  */
-module.exports = function(cache) {
-  cache = cache || {};
```

Comment:
Removing this cache makes `babel-loader` got to the file-system for every file that it compiles, which seems not to be a good idea.
Or can you explain the reason for this change?